pkg/schedule/operator/waiting_operator.go

```diff
@@ -26,6 +26,7 @@ var priorityWeight = []float64{1.0, 4.0, 9.0, 16.0}
 // WaitingOperator is an interface of waiting operators.
 type WaitingOperator interface {
 	PutOperator(op *Operator)
+	PutMergeOperators(op []*Operator)
```

Comment:
How about:
```suggestion
	PutOperators(op []*Operator)
```

Reply:
Now it is only used for merge, and the common name is easy to misunderstand.